src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -5,65 +5,77 @@
 
 import java.util.Optional;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
 
 @DataJpaTest
 public class AnswerRepositoryTest {
-	private final User JAVAJIGI = new User(1L, "javajigi", "password", "name", "javajigi@slipp.net");
-	private final User SANJIGI = new User(2L, "sanjigi", "password", "name", "sanjigi@slipp.net");
-	private final Question Q1 = new Question("title1", "contents1").writeBy(JAVAJIGI);
+	private User javajigi;
+	private User sanjigi;
+	private Question question;
+	private Answer expected1;
+	private Answer expected2;
 
 	@Autowired
 	private AnswerRepository answerRepository;
 
+	@Autowired
+	private QuestionRepository questionRepository;
+
+	@Autowired
+	private UserRepository userRepository;
+
+	@BeforeEach
+	void init() {
```

Comment:
```suggestion
	void init(@Autowired final UserRepository userRepository) {
```

init 단계에서만 사용하는 빈의 범위를 줄여볼 수 있겠네요 ㅎㅎ

Reply:
음!! 더 깔끔하게 정리되네요. 감사합니다 = )